tests/modules/hub.py

```diff
@@ -4,9 +4,16 @@
 import torchvision.models as models
 import timm
 from transformers import BertModel, BertTokenizer, BertConfig
+import os
+import json
 
 torch.hub._validate_not_a_forked_repo = lambda a, b, c: True
 
+torch_version = torch.__version__
+
+# Downloads all model files again if manifest file is not present
+MANIFEST_FILE = 'model_manifest.json'
+
 models = {
     "alexnet": {
```

Comment:
So for example you can add 
```
    "loop_fallback_eval": {
        "model": LoopFallbackEval(),
        "path": "script",
     },
} 
```

Reply:
@narendasan : I added the same. I have segregated the custom model file from hub.py.
I am not sure about where to put the Bert-base-uncased case